include/seqan3/alignment/pairwise/policy/scoring_scheme_policy.hpp

```diff
@@ -58,6 +60,33 @@ class scoring_scheme_policy
 
     //!\brief The scoring scheme used for this alignment algorithm.
     scoring_scheme_t scoring_scheme{};
+
+    /*!\brief Maybe converts the given sequence value to a specific profile used by the underlying scoring scheme.
+     *
+     * \details
+     *
+     * In the vectorised alignment the scoring scheme might transform the sequence values of the first sequence into
+     * a profile for a more efficient comparison of the sequence characters in simd mode.
+     *
+     * If the given sequence type models seqan3::semialphabet the function becomes a no-op function and returns the
+     * unmodified value.
+     */
+    template <typename sequence1_value_t>
+    //!\cond
+        requires simd_concept<remove_cvref_t<sequence1_value_t>>
+    //!\endcond
+    auto scoring_scheme_profile_column(sequence1_value_t && sequence1_value) const noexcept
```

Comment:
```suggestion
    auto scoring_scheme_profile_column(alphabet_t && alphabet) const noexcept
```

Like in `policy_scoring_scheme.hpp`